src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -22,8 +22,8 @@
   private TalonSRX shooterMaster = new TalonSRX(Constants.SHOOTER_SHOOTER_MASTER_ID);
   private VictorSPX shooterFollower = new VictorSPX(Constants.SHOOTER_SHOOTER_FOLLOWER_ID);
 
-  private int setPoint = 135000;
-  private int closeSetpoint = 125000;
+  private int setPoint = 75000;
+  private int closeSetpoint = 85000;
```

Comment:
Sam, 75000 will be our primary shot and 85000 will be our control panel shot (we may not use our control panel shot)